### ISSUE TITLE
Cannot build on rust 1.70.0

### ISSUE THREAD

**Issue by yorkz1994** [honzasp/makiko#2]
1. Upgrade rust to latest release 1.70.0
2. Clone makiko or use it as dependency.
3. Cargo build, errors:
```
error: `box_syntax` has been removed
   --> /home/york/.cargo/registry/src/index.crates.io-6f17d22bba15001f/guard-0.5.1/src/lib.rs:426:20
    |
426 |         let foo = (box 42, [1, 2, 3]);
    |                    ^^^^^^
    |
help: use `Box::new()` instead
    |
426 |         let foo = (Box::new(42), [1, 2, 3]);
    |                    ~~~~~~~~~~~~

```

**Comment by honzasp** [comment]
Thank you for reporting this! I fixed this by replacing the `guard` crate with native `let else` statements and I will release a new version today.

**Comment by honzasp** [comment]
Fixed in version 0.2.2

**Comment by yorkz1994** [comment]
Impressive! Thank you for the quick fix.
